src/comm.cpp

```diff
@@ -1536,9 +1533,9 @@ void act_new(const char *format, CHAR_DATA *ch, const void *arg1, const void *ar
             Descriptor *d;
 
             for (d = descriptor_list; d != nullptr; d = d->next) {
-                if (d->character != nullptr && d->character->in_room != nullptr && d->character->in_room->vnum == 1222
-                    && IS_AWAKE(d->character))
-                    send_to_char(buf, d->character);
+                if (d->character() && d->character()->in_room != nullptr && d->character()->in_room->vnum == 1222
```

Comment:
we should probably bundle all these magic room number constants into one place with constexprs. I know there's a few of them.

Reply:
Bloody right.